sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2422,6 +2423,9 @@ class ReadFromBigQuery(PTransform):
       to run queries with INTERACTIVE priority. This option is ignored when
       reading from a table rather than a query. To learn more about query
       priority, see: https://cloud.google.com/bigquery/docs/running-queries
+    output_type (str): By default, the schema returned from this transform
+    would be of type TableSchema. Other schema types can be specified
+    ("BEAM_SCHEMAS").
```

Comment:
This argument should be referring to the type of the elements in the output PCollection. The status quo (what should become the default) is to produce Python `dict`s, let's call this `"PYTHON_DICT"`, and make that default rather than `None`.

For this new feature, I think we should call it `"BEAM_ROW"` to standardize with other terminology across Beam (a *Row* is an instance/element, whose type is described by a *Schema*)